dali/operators/geometric/CMakeLists.txt

```diff
@@ -15,4 +15,4 @@
 # Get all the source files and dump test files
 collect_headers(DALI_INST_HDRS PARENT_SCOPE)
 collect_sources(DALI_OPERATOR_SRCS PARENT_SCOPE)
-collect_test_sources(DALI_TEST_SRCS PARENT_SCOPE)
```

Comment:
By any chance, can you add new lines when they are missing in CMakeLists.txt?

Reply:
What would be the benefit of that?